template.yaml

```diff
@@ -219,6 +227,18 @@ Resources:
       Role: !GetAtt LambdaEdgeExecutionRole.Arn
       Timeout: 5
 
+  DefaultPermissionBoundaryPolicy:
+    Type: AWS::IAM::ManagedPolicy
+    Condition: CreateDefaultBoundaryPolicy
+    Properties:
+      Description: "non-restrictive default permission boundary, used only if no separate PermissionBoundaryPolicyArn is used"
+      PolicyDocument:
+        Version: "2012-10-17"
+        Statement:
+          - Effect: Allow
+            Action: '*'
+            Resource: '*'
```

Comment:
should we not create this policy using terraform to control AWS resources  or are we managing the roles and policies for the organization

I guess this is just a policy that we can use in other templates as well (Please correct me if I am wrong )

Reply:
This is a default boundary permission doing nothing, basically, so that it's backward compatible for consumers of [the original repo ](https://github.com/aws-samples/cloudfront-authorization-at-edge) who don't have to bother with boundary permissions.

This default policy can then be overriden using the new variable, in BG's case